smoke-test/pom.xml

```diff
@@ -26,7 +26,7 @@
     <testResources>
       <testResource>
         <directory>src/test/resources</directory>
-        <filtering>true</filtering>
+        <filtering>false</filtering>
```

Comment:
Is there any need to change this ?

Reply:
This one yes, I have to change as the jetty.keystore should be copied in binary mode and file filtering change the file during copy. After that change the file becomes unreadable.